src/public/navigation.ts

```diff
@@ -13,7 +13,10 @@ import { sendMessageToParent } from '../internal/communication';
  * @param navigateForward Determines the direction to focus in teams app.
  */
 export function returnFocus(navigateForward?: boolean): void {
-  ensureInitialized(FrameContexts.content);
+  ensureInitialized(
```

Comment:
Please change this to `ensureInitialized();`  since we will eventually want this API to work in all frame contexts.

Reply:
Done